tests/layer_tests/tensorflow_tests/test_tf_Add.py

```diff
@@ -222,3 +222,58 @@ def test_add_placeholder_const_broadcast_5D(self, params, ie_device, precision,
                                                           use_legacy_frontend=use_legacy_frontend),
                    ie_device, precision,
                    ir_version=ir_version, temp_dir=temp_dir, use_legacy_frontend=use_legacy_frontend)
+
+
+class TestComplexAdd(CommonTFLayerTest):
+        def _prepare_input(self, inputs_info):
+            rng = np.random.default_rng()
+            assert 'param_real_1:0' in inputs_info
+            assert 'param_imag_1:0' in inputs_info
+            assert 'param_real_2:0' in inputs_info
+            assert 'param_imag_2:0' in inputs_info
+            param_real_shape_1 = inputs_info['param_real_1:0']
+            param_imag_shape_1 = inputs_info['param_imag_1:0']
+            param_real_shape_2 = inputs_info['param_real_2:0']
+            param_imag_shape_2 = inputs_info['param_imag_2:0']
+            inputs_data = {}
+            inputs_data['param_real_1:0'] = 4 * rng.random(param_real_shape_1).astype(np.float32) - 2
+            inputs_data['param_imag_1:0'] = 4 * rng.random(param_imag_shape_1).astype(np.float32) - 2
+            inputs_data['param_real_2:0'] = 4 * rng.random(param_real_shape_2).astype(np.float32) - 2
+            inputs_data['param_imag_2:0'] = 4 * rng.random(param_imag_shape_2).astype(np.float32) - 2
+            return inputs_data
+
+        def create_complex_addv2_net(self, input_shape):
+            import tensorflow as tf
+            tf.compat.v1.reset_default_graph()
+            # Create the graph and model
+            with tf.compat.v1.Session() as sess:
+                param_real1 = tf.compat.v1.placeholder(np.float32, input_shape, 'param_real_1')
+                param_imag1 = tf.compat.v1.placeholder(np.float32, input_shape, 'param_imag_1')
+                param_real2 = tf.compat.v1.placeholder(np.float32, input_shape, 'param_real_2')
+                param_imag2 = tf.compat.v1.placeholder(np.float32, input_shape, 'param_imag_2')
+                complex1 = tf.raw_ops.Complex(real=param_real1, imag=param_imag1)
+                complex2 = tf.raw_ops.Complex(real=param_real2, imag=param_imag2)
+                add = tf.raw_ops.AddV2(x=complex1, y=complex2, name="complex_add")
```

Comment:
here, you also need to add `Real` and `Imag` operations since OV does not support complex type on outputs. We only assume support of complex tensors between intermediate layers.

Reply:
@rkazants thanks for your input. I will address the changes